ui-design-lib/src/main/java/co/electriccoin/zcash/ui/design/component/ZashiBigIconButton.kt

```diff
@@ -91,21 +92,25 @@ fun ZashiBigIconButton(
         shadowElevation = shadowElevation
     ) {
         Column(
-            modifier = backgroundModifier.padding(16.dp),
+            modifier = backgroundModifier,
             horizontalAlignment = Alignment.CenterHorizontally,
             verticalArrangement = Arrangement.Center
         ) {
             Image(
+                modifier = Modifier.padding(start = 16.dp, top = 16.dp, end = 16.dp),
                 painter = painterResource(state.icon),
                 contentDescription = state.text.getValue(),
                 colorFilter = ColorFilter.tint(ZashiColors.Text.textPrimary)
             )
             Spacer(Modifier.height(4.dp))
             Text(
```

Comment:
We could use our existing `AutoSizingText` component here. However, it might need improvement before being used.